lib/events.js

```diff
@@ -416,8 +423,10 @@ EventEmitter.prototype.removeListener =
 
         if (position === 0)
           list.shift();
-        else
-          spliceOne(list, position);
+        else {
+          lazyUtil();
+          util.spliceOne(list, position);
```

Comment:
Why not just use `list.splice(position, 1)` here?
This was introduced 3 years ago in https://github.com/nodejs/node/commit/d3f8db124463e478420d8bba6fa5cf13af7b4ecb

Reply:
According to https://github.com/nodejs/node/pull/14082, it seems like `Array#splice` is still much slower than `spliceOne`.

Reply:
Than I'd rather have the simpler lazy loader, like https://github.com/nodejs/node/pull/14167
```js
var spliceOne;
...
if (!spliceOne) spliceOne = require('internal/util').spliceOne;
```